float64 heading
float64 start_x
float64 start_y
float64 end_x
float64 end_y
string lane_type